test-configs.yaml

```diff
@@ -160,6 +160,72 @@ test_plans:
             - 'defconfig+virtualvideo'
             - 'multi_v7_defconfig+virtualvideo'
 
+  uefi-arm64-gicv2:
+    rootfs: buildroot_ramdisk
+    params:
+        ctx_gicversion: "gic-version=2"
+        ctx_cpu: "cortex-a57"
```

Comment:
The `ctx_cpu` is already defined in the template, it's derived from the CPU architecture.  And it doesn't seem like it needs to be set to anything different than what is already in `generic-qemu-boot-template.jinja2`, so I think it should be dropped here.